repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/publication-partial-info.ftl

```diff
@@ -0,0 +1,46 @@
+<#macro publicationPartialInfo publicationListingPageURL globalPublicationTypeMap globalProfessionsMap globalTopicsMap>
+<#--  Publication type  -->
+    <#if globalPublicationTypeMap?has_content>
+        <div class="hee-card--details__item">
+            <span>Publication type:</span>
+            <#if publicationListingPageURL?has_content>
+                <a href=${publicationListingPageURL}?publicationType=${globalPublicationTypeMap.taxonomyValues[0].key}>${globalPublicationTypeMap.taxonomyValues[0].label}</a>
+            <#else>
+                ${globalPublicationTypeMap.taxonomyValues[0].label}
+            </#if>
+        </div>
+    </#if>
+
+<#--  Publication professions  -->
+    <#if globalProfessionsMap?has_content>
```

Comment:
We may need to rewrite this into `<#if globalProfessionsMap?? && globalProfessionsMap.taxonomyValues?has_content>` to handle scenarios wherein `globalTaxonomyProfessions` field is empty.

Reply:
Now done